File: verification/council/fallback.go
package council

import "context"

type ManualFallbackProvider struct{}

func NewManualFallbackProvider() *ManualFallbackProvider {
	return &ManualFallbackProvider{}
}

func (p *ManualFallbackProvider) Verify(ctx context.Context, registrationNumber string, council string) (*CouncilVerificationResult, error) {
	return &CouncilVerificationResult{
		IsVerified:         false,
		RegistrationNumber: registrationNumber,
		CouncilName:        council,
		Status:             "MANUAL_REVIEW_REQUIRED",
		VerificationSource: "MANUAL_FALLBACK",
		Remarks:            "Official online API unavailable for council. Marked for manual review.",
	}, nil
}

func (p *ManualFallbackProvider) GetProviderName() string {
	return "MANUAL_FALLBACK_PROVIDER"
}

File: python_backend/verification/council/fallback.py
from .provider import CouncilVerificationProvider, CouncilVerificationResult

class ManualFallbackProvider(CouncilVerificationProvider):
    def verify(self, ctx: dict, registration_number: str, council: str) -> CouncilVerificationResult:
        return CouncilVerificationResult(
            is_verified=False,
            registration_number=registration_number,
            council_name=council,
            status="MANUAL_REVIEW_REQUIRED",
            verification_source="MANUAL_FALLBACK",
            remarks="Official online API unavailable for council. Marked for manual review."
        )

    def get_provider_name(self) -> str:
        return "MANUAL_FALLBACK_PROVIDER"
